Frameworks/UIKit/UIImage.mm

```diff
@@ -889,22 +892,34 @@ + (UIImage*)animatedResizableImageNamed:(NSString*)name capInsets:(UIEdgeInsets)
     return StubReturn();
 }
 
+static NSData* _ObtainImageRepresentation(UIImage* img, CFStringRef type, NSDictionary* properties) {
```

Comment:
I would _prefer_ CreateImageRepresentation, and it would return a +1.
Then, UIImageXXXRepresentation could cast it and autorelease it.

I'd like to avoid using CFAutorelease wherever possible because it goes through the objective-C runtime and vends objects out to the local thread's autorelease pool.

Using woc::AutoCF or even woc::unique_cf is *infinitely preferable* to manually managing these lifetimes by vending them to global state.